R/tm_g_bivariate.R

```diff
@@ -370,7 +373,7 @@ srv_g_bivariate <- function(id,
 
     anl_merged_q <- reactive({
       req(anl_merged_input())
-      teal.code::new_qenv(tdata2env(data), code = get_code(data)) %>%
+      teal.code::new_qenv(tdata2env(data), code = teal.data::get_code(data)) %>%
```

Comment:
So get_code.tdata lives in teal so do we need `teal.data::` here? Would be really sad if we do...

Reply:
https://github.com/insightsengineering/teal/pull/739